server/controllers/mixedWebAndPdfMemosCtrl.js

```diff
@@ -4,28 +4,32 @@ const log = require('@kth/log')
 const { StoredMemoPdfsModel } = require('../models/storedMemoPdfsModel')
 const dbArrayOfDocument = require('../lib/dbSeveralDocument')
 
-const mergeRoundsWithTheSamePdfFile = (miniFileInfo, koppsRoundId) => {
-  const { ladokRoundIds } = miniFileInfo
-  return [...ladokRoundIds, koppsRoundId].sort((a, b) => Number(a) - Number(b))
+const mergeApplicationCodesWithTheSamePdfFile = (miniFileInfo, applicationCode) => {
+  const { applicationCodes } = miniFileInfo
+  return [...applicationCodes, applicationCode].sort((a, b) => Number(a) - Number(b))
 }
 
 const pdfMemosTree = dbMigratedPdfs => {
   const pdfMemos = {}
-  dbMigratedPdfs.map(({ courseCode, courseMemoFileName, koppsRoundId, semester, lastChangeDate }) => {
-    if (!koppsRoundId || koppsRoundId === '') return
+  dbMigratedPdfs.map(({ courseCode, courseMemoFileName, koppsRoundId, semester, lastChangeDate, applicationCode }) => {
+    if (!applicationCode || applicationCode === '') return
 
     const semesterInfo = pdfMemos[semester] || {}
     const miniFileInfo = semesterInfo[courseMemoFileName] || {}
-    const { ladokRoundIds } = miniFileInfo
-    if (ladokRoundIds) {
-      pdfMemos[semester][courseMemoFileName].ladokRoundIds = mergeRoundsWithTheSamePdfFile(miniFileInfo, koppsRoundId)
+    const { applicationCodes } = miniFileInfo
+    if (applicationCodes) {
+      pdfMemos[semester][courseMemoFileName].applicationCodes = mergeApplicationCodesWithTheSamePdfFile(
+        miniFileInfo,
+        applicationCode
+      )
     } else {
       pdfMemos[semester] = {
         ...pdfMemos[semester],
         [courseMemoFileName]: {
           courseCode,
           courseMemoFileName,
           ladokRoundIds: [koppsRoundId],
```

Comment:
`ladokRoundIds: [koppsRoundId]`, has to be removed

Reply:
Done